definitions/procedures/backup/config_files.rb

```diff
@@ -33,7 +36,12 @@ def run
 
         puts "WARNING: Attempt #{attempt_no}/#{MAX_RETRIES} to collect all config files failed!"
         puts 'Some files were modified during creation of the archive.'
-        if attempt_no == MAX_RETRIES
+        if attempt_no == MAX_RETRIES && @exclude_qpidd_data
+          tar_cmd = tar_command(configs, to_exclude.push('/var/lib/qpidd'))
+          with_spinner('Collecting config files to backup, skipping /var/lib/qpidd') do
```

Comment:
As offline discussion, it would be good to have warning instead of spinner showing this information.